evm/src/generation/mod.rs

```diff
@@ -41,6 +43,10 @@ pub struct GenerationInputs {
     /// storage tries, and nodes therein, that will be accessed by these transactions.
     pub storage_tries: Vec<(Address, PartialTrie)>,
 
+    /// Mapping between smart contract code hashes and the contract byte code.
+    /// All account entries that have `code_hash` filled in will have their corresponding byte code in this map.
```

Comment:
I don't think this is strictly necessary - say a certain account receives tokens, then our zkEVM will need the its data `(nonce, balance, codeHash, storageRoot)` to prove that `balance` was updated properly, but we won't need its code

Reply:
Oh true. I'll update the comment.